app/navigation/navigation-core/src/main/kotlin/com/hedvig/android/navigation/core/TopLevelGraph.kt

```diff
@@ -1,39 +1,9 @@
 package com.hedvig.android.navigation.core
 
-import com.kiwi.navigationcompose.typed.Destination
-import kotlinx.serialization.Serializable
-
-sealed interface TopLevelGraph : Destination {
-  @Serializable
-  data object HOME : TopLevelGraph
-
-  @Serializable
-  data object INSURANCE : TopLevelGraph
-
-  @Serializable
-  data object FOREVER : TopLevelGraph
-
-  @Serializable
-  data object PROFILE : TopLevelGraph
-
-  fun toName(): String {
-    return when (this) {
-      HOME -> "HOME"
-      INSURANCE -> "INSURANCE"
-      FOREVER -> "REFERRALS"
-      PROFILE -> "PROFILE"
-    }
-  }
-
-  companion object {
-    fun fromName(input: String): TopLevelGraph? {
-      return when (input) {
-        "HOME" -> HOME
-        "INSURANCE" -> INSURANCE
-        "REFERRALS" -> FOREVER
-        "PROFILE" -> PROFILE
-        else -> null
-      }
-    }
-  }
+enum class TopLevelGraph {
```

Comment:
I guess they shouldn't be serializable now?

Reply:
They are no longer a `Destination` but just an enum to enumerate all the possible graphs we got. We use this as a way to be able to exhaustively go over them inside HedvigAppState for example. Like here
https://github.com/HedvigInsurance/android/blob/afb510b3ebdf48ebc9e9597d31fc5de13d327081/app/app/src/main/kotlin/com/hedvig/android/app/ui/IsTopLevelGraphInHierarchy.kt#L19-L23
or here
https://github.com/HedvigInsurance/android/blob/e6a57dc642bf66349834d4ed219cebe9b78ba4b8/app/navigation/navigation-ui/src/main/kotlin/com/hedvig/android/navigation/ui/TopLevelGraphExt.kt#L2-L1
or here
https://github.com/HedvigInsurance/android/blob/14eaa9cbb8cbe59d074790c62e754b5fc7738460/app/app/src/main/kotlin/com/hedvig/android/app/ui/HedvigAppState.kt#L162
etc.

The graph destination itself which the navigation library is using is a different type, like here https://github.com/HedvigInsurance/android/blob/df388b81b92baa091a692146b0558a524424a767/app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/navigation/ProfileDestinations.kt#L9

This allows those to just stay inside their modules without having to be available everywhere like they used to in AppDestinations, but we still want to be able to exhaustively go over them, hence this enum.